mayastor/src/bdev/dev/nvmf.rs

```diff
@@ -176,6 +181,8 @@ impl CreateDestroy for Nvmf {
                 name: self.name.clone(),
             })?;
 
+        uri::dev_set_uuid(&self.name, self.uuid);
```

Comment:
the bdev created, will already have a UUID defined by the target, so no point in setting it.